reader-privacy-addendum.md

```diff
@@ -0,0 +1,30 @@
+# Reader Privacy Implementation Details
```

Comment:
Can we move this to a single doc, under `Implementation/AESGSM`or `Implementers Considerations/AESGSM` section please?

Reply:
Done